bindings_ffi/src/mls.rs

```diff
@@ -534,6 +534,32 @@ impl FfiXmtpClient {
             scw_verifier: self.inner_client.scw_verifier().clone().clone(),
         }))
     }
+
+    /**
+     * Revoke a list of installations
+     */
+    pub async fn revoke_installations(
+        &self,
+        installation_ids: Vec<Vec<u8>>,
+    ) -> Result<Arc<FfiSignatureRequest>, GenericError> {
+        let installation_id = self.inner_client.installation_public_key();
+
+        if installation_ids.iter().any(|id| id == &installation_id) {
+            return Err(GenericError::Generic {
+                err: "Cannot revoke the current installation ID.".to_string(),
+            });
+        }
```

Comment:
@neekolas any preference here should we error if they try to revoke the installation they are currently logged into or should we let users revoke that installation if they want to?

Reply:
I think there should be some way to revoke your current installation, even if it likely will break things in your current app until you create a new client.

Reply:
At the very least, I don't think we should block it at the protocol level even if it's a bad idea in most cases.